package.json

```diff
@@ -165,6 +165,7 @@
     "p-memoize": "^7.1.1",
     "p-queue": "^8.0.1",
     "p-ratelimit": "^1.0.1",
+    "polished": "^4.3.1",
```

Comment:
Don't think adding 2.93 MB is worth it for a single use-case https://packagephobia.com/result?p=polished
```typescript
import { lighten } from "@chakra-ui/theme-tools";
```

Though it's deprecated and could be removed anytime we update chakra-ui, but until then, we could use it